src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -449,6 +449,19 @@ s! {
         pub kve_path: [[::c_char; 32]; 32],
     }
 
+    pub struct kinfo_kstack {
+        pub kkst_tid: ::lwpid_t,
+        pub kkst_state: ::c_int,
+        pub kkst_trace: [[::c_char; 32]; 32],
```

Comment:
I'd rather keep the original size and just use a `cfg()` to only make this type available on newer rustc versions.